pyproject.toml

```diff
@@ -1,51 +1,66 @@
-[project]
-authors = [
-    { name = "Dylan Anthony", email = "contact@dylananthony.com" },
-    { name = "Benchling Support", email = "support@benchling.com" },
-]
-license = { text = "MIT" }
-requires-python = ">=3.8.1,<4.0"
-dependencies = [
-    "jinja2>=3.0.0,<4.0.0",
-    "typer>0.6,<0.13",
-    "colorama>=0.4.3; sys_platform == \"win32\"",
-    "shellingham>=1.3.2,<2.0.0",
-    "pydantic>=2.1.1,<3.0.0",
-    "attrs>=21.3.0",
-    "python-dateutil>=2.8.1,<3.0.0",
-    "httpx>=0.20.0,<0.28.0",
-    "ruamel.yaml>=0.18.6,<0.19.0",
-    "ruff>=0.2,<0.8",
-    "typing-extensions>=4.8.0,<5.0.0",
-]
-name = "openapi-python-client"
+# This fork of openapi-python-client is published internally at Benchling. Changes in this
+# project file will *not* be contributed upstream.
+
+[tool.poetry]
+name = "benchling-openapi-python-client"
 version = "2.0.0-alpha.1"
-description = "Generate modern Python clients from OpenAPI"
-keywords = [
-    "OpenAPI",
-    "Client",
-    "Generator",
+description = "Generate modern Python clients from OpenAPI - Benchling fork"
+repository = "https://github.com/benchling/openapi-python-client"
+license = "MIT"
+keywords=["OpenAPI", "Client", "Generator"]
+authors = [
+    "Dylan Anthony <danthony@triaxtec.com>",
+    "Benchling Support <support@benchling.com>",
 ]
 classifiers = [
-    "Development Status :: 4 - Beta",
-    "License :: OSI Approved :: MIT License",
+    "Development Status :: 3 - Alpha",
     "Intended Audience :: Developers",
-    "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3.8",
-    "Programming Language :: Python :: 3.9",
-    "Programming Language :: Python :: 3.10",
-    "Programming Language :: Python :: 3.11",
-    "Programming Language :: Python :: 3.12",
-    "Programming Language :: Python :: 3.13",
     "Topic :: Software Development :: Code Generators",
     "Typing :: Typed",
 ]
 readme = "README.md"
+packages = [
+    {include = "openapi_python_client"},
+    # Note that the module name is different from the package name, so that we can
+    # transparently switch between the upstream package and this one in our projects.
+]
+include = [
+    "CHANGELOG.md",
+    "openapi_python_client/py.typed",
+]
+exclude = [
+    ".gitignore",
+]
 
-[project.urls]
-repository = "https://github.com/openapi-generators/openapi-python-client"
-
-[project.scripts]
+[tool.poetry.dependencies]
```

Comment:
The dependencies in this section are exactly equivalent to the ones that the upstream package was declaring with `pdm`.